api/src/test/resources/org/openmrs/api/db/hibernate/include/HibernateAlertDAOTestDataSet.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<dataset>
+	<notification_alert alert_id="2" creator="1" text="Testing hibernate Dao" date_created="2019-01-01 00:00:00.0"  date_to_expire="2020-08-08 00:00:00.0" satisfied_by_any="1" alert_read="1" uuid="16914264-e162-11df-9195-001e378eb69e"/>
```

Comment:
@dkayiwa @ibacher , for this pr,i have one method which is failing locally ie the delete method whose snipet is:  @Test
	public void deleteAlert_shouldReturnNullAfterDeleting() {
		Alert savedAlert = hibernateAlertDAO.getAlert(2);
		Assert.assertNotNull(savedAlert);
		hibernateAlertDAO.deleteAlert(savedAlert);
		Assert.assertNull(savedAlert);
	}
it returns a 2 instead of a null,on the assertNull section,any thoughts?

Reply:
You probably need to refresh the `savedAlert` after deleting it from the database (it isn't going to be magically rendered `null`).

```java
public void deleteAlert_shouldReturnNullAfterDeleting() {
	Alert savedAlert = hibernateAlertDAO.getAlert(2);
	assertNotNull(savedAlert);

	hibernateAlertDAO.deleteAlert(savedAlert);

	assertNull(hibernateAlertDAO.getAlert(2));
}
```